### PR TITLE
Update `font-family-name-quotes` to also check `font` properties

### PR DESCRIPTION
**Pull request by m-allanson**
Refs #1398

### CODE REVIEW

File: src/rules/font-family-name-quotes/__tests__/index.js [hunk 1]
```diff
@@ -30,6 +30,12 @@ testRule(rule, {
   accept: variablePositiveTests.concat([ {
     code: "a { font-family: \"Lucida Grande\", \"Arial\", sans-serif; }",
   }, {
+    code: "a { font: 1em \"Lucida Grande\", \"Arial\", sans-serif; }",
```
Comment (jeddy3) [inline src/rules/font-family-name-quotes/__tests__/index.js]: Could we get a passing test that shows that a property like `fonts` (with the s on the end) is defo ignored please?

File: src/rules/font-family-name-quotes/README.md [hunk 2]
```diff
@@ -8,6 +8,8 @@ a { font-family: "Times New Roman", 'Ancient Runes', serif; }
  *               These quotation marks and this one */
 ```
 
+This rule checks the `font` and `font-family properties`.
```
Comment (jeddy3) [inline src/rules/font-family-name-quotes/README.md]:
```md
This rule checks the `font` and `font-family` properties.
```

**Comment by jeddy3** [comment]
Looking good! I have one request and one typo fix for you.

**Comment by evilebottnawi** [comment]
LGTM, some fixes above

**Comment by m-allanson** [comment]
@jeddy3 Good catches, updated.

**Comment by jeddy3** [comment]
Thanks for the amends. Looks great.

**Pull request merged**